skyportal/handlers/api/sources.py

```diff
@@ -2445,6 +2447,45 @@ async def get_sources(
                     log_verbose(
                         f"17. Color Mag Query took {endTime - startTime} seconds."
                     )
+            if include_tags:
+                startTime = time.time()
+
+                query_str, bindparams = array2sql(
+                    obj_ids, type=sa.String, prefix="obj_ids"
+                )
+                stmt = f"""
+                SELECT obj_tags.obj_id, obj_tags.id, objtagoptions.name, obj_tags.objtagoption_id
+                FROM obj_tags
+                JOIN objtagoptions ON obj_tags.objtagoption_id = objtagoptions.id
+                WHERE obj_tags.obj_id IN {query_str}
+                """
+
+                tags_result = session.execute(text(stmt).bindparams(*bindparams))
+                tags_by_obj = {}
+
+                for row in tags_result:
+                    obj_id = row[0]
```

Comment:
Would be more elegant to deconstruct this with smth like:
```python
obj_id, tag_id, tag_name, objtagoption, _ = row
```
or similar, not that you should make any change here but just good to keep in mind for later